scripts/patch-release/patch-release.go

```diff
@@ -55,45 +58,71 @@ func run() error {
 		return fmt.Errorf("open local repo: %w", err)
 	}
 
-	currentBranch, err := repo.CurrentBranch()
-	if err != nil {
-		return fmt.Errorf("get current branch: %w", err)
+	for _, minorVersion := range releaseMinorVersions {
+		baseBranchName := branchPrefix + minorVersion // returns "release-x.xx"
```

Comment:
technically it should be
```suggestion
		baseBranchName := branchPrefix + minorVersion // returns "release-x.y"
```